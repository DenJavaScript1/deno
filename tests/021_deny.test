args: tests/021_deny.ts --deny
check_stderr: true
exit_code: 1
output: tests/021_deny.ts.out
