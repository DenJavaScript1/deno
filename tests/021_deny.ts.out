[WILDCARD]PermissionDenied: permission denied
    at DenoError (deno/js/errors.ts:19:5)
    at maybeError (deno/js/errors.ts:38:12)
    at maybeThrowError (deno/js/errors.ts:26:15)
    at sendSync (deno/js/dispatch.ts:67:5)
    at listen (deno/js/net.ts:147:19)[WILDCARD]
